src/export-to-csv.spec.ts

```diff
@@ -1,4 +1,5 @@
-import { ExportToCsv, Options } from './export-to-csv';
+import { ExportToCsv } from './export-to-csv';
```

Comment:
I wonder how those tests passed successfully w/o using the 'keys' property at all... or, did they? Please fix the tests after merging the PR.